core/templates/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -130,25 +133,29 @@ export class Exploration {
   }
 
   getInteractionCustomizationArgs(
-      stateName: string): InteractionCustomizationArgs {
+      stateName: string): InteractionCustomizationArgs | null {
     let interaction = this.getInteraction(stateName);
     if (interaction === null) {
       return null;
     }
     return interaction.customizationArgs;
   }
 
-  getInteractionInstructions(stateName: string): string {
+  getInteractionInstructions(stateName: string): string | null {
```

Comment:
Why can this be `null`? Below you return either instructions or `''`

Reply:
The 'instructions' key is null for some objects in the interaction specs constant. Hence the return type is string | null.

Reply:
That is a bit weird to return `''` when the `interactionId` is `null`. Also I tried looking through the codebase and I don't seem to find any usages of `getInteractionInstructions` except for the spec files, can you please try to confirm and delete this function if it is not used?

Reply:
Yes you are right, the functions is not being used anywhere. Removed it.